### DISCUSSION TITLE
m estimation: where does the initial decision rule (match/not match) come from?

### DISCUSSION THREAD

**Discussion by nadd0u** [moj-analytical-services/splink#1978] [Q&A] [answered]
Hello!  

When estimating `m` the P(observation γ|match) , how does splink determine which record pairs are matches? From my understanding the Expecation-Maximization algorithm  starts with a rough decision rule to identify matches and non matches and then iterates to update that rule? Where does this initial decision boundary come from?  

I'm asking because my model is identifying pairs as matches even when first name is completely different &  I'm concerned that its because it is inappropriately defining matches when estimating `m`.

**Comment by RobinL** [comment, marked as answer]
There are starting values of the m probabilities built into Splink.  They can be overridden by manually specifying them in your settings object.  But I think the problem you're seeing is less likely to be down to the wrong starting values for m probabilities, and more likely to be down to other issues with your data or model spec, possibly strong correlations between your input columns causing violations of conditional independence, resulting in double counting and hence causing problems with the EM approach.
